modules/drivers/video/BUILD

```diff
@@ -0,0 +1,43 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "libvideo_driver_component.so",
+    linkopts = ["-shared"],
+    linkstatic = False,
+    deps = [":video_driver_component_lib"],
+)
+
+cc_library(
+    name = "video_driver_component_lib",
+    srcs = ["video_driver_component.cc"],
+    hdrs = ["video_driver_component.h"],
+    copts = ['-DMODULE_NAME=\\"video\\"'],
+    deps = [
+        "//cyber",
+        "//modules/common/util:message_util",
+        "//modules/drivers/video:video_lib",
+    ],
+)
+
+cc_library(
+    name = "video_lib",
+    srcs = [
+        "driver.cc",
+        "socket_input.cc",
+    ],
+    hdrs = [
+        "driver.h",
+        "input.h",
+        "socket_input.h",
```

Comment:
Please split them to at least two cc_library, `driver` and `socket_input`

Reply:
agree, but for the header file, driver.cc must include socket_input.h， is it ok?

Reply:
Yeah, it's OK. Just add the `socket_input` cc_library as dependency, instead of adding it to the `hdrs`.